src/main/java/io/jenkins/plugins/restlistparam/RestListParameterDefinition.java

```diff
@@ -71,6 +74,7 @@ public RestListParameterDefinition(final String name,
     this.mimeType = mimeType;
     this.valueExpression = valueExpression;
     this.credentialId = StringUtils.isNotBlank(credentialId) ? credentialId : "";
+    this.displayExpression = displayExpression;
```

Comment:
```suggestion
    this.displayExpression = StringUtils.isNotBlank(displayExpression) ? displayExpression : "$";
```
To prevent breakage due to NPEs this new field has to be set to a default value if its null, otherwise you would have to go into each pilelines config using the param and hit save to fix this issue. (FYI from something I once broke and learned the hard way xD)